src/test/java/com/salesforce/storm/spout/dynamic/VirtualSpoutTest.java

```diff
@@ -79,6 +80,14 @@
  */
 public class VirtualSpoutTest {
 
+    /**
+     * Before each test reset the static on the MockConsumer.
+     */
+    @BeforeEach
+    void resetMockConsumerPartitions() {
+        MockConsumer.partitions = Arrays.asList(1);
```

Comment:
@Crim this ended up being my flapper problem. Some day I'll revisit the `MockConsumer` entirely and fix this in a better way. You called it though. 😉